src/components/index/LiquidityPoolList.tsx

```diff
@@ -54,9 +53,9 @@ function LiquidityPoolCard (
   const SymbolBIcon = getAssetIcon(props.tokenBSymbol)
   const SymbolAIcon = getAssetIcon(props.tokenASymbol)
   return (
-    <div className='flex p-4 md:p-6 border border-gray-300 h-30'>
-      <div className='flex flex-col justify-between my-auto w-1/4'>
-        <div className='flex icons'>
+    <div className='flex flex-col p-4 md:p-6 border border-gray-300 h-30 space-y-3'>
+      <div className='flex items-center space-x-1 my-auto'>
+        <div className='flex icons transform rotate-45'>
           <SymbolAIcon className='h-6 w-6 z-10' />
           <SymbolBIcon className='h-6 w-6 -ml-2' />
```

Comment:
```suggestion
        <div className='flex icons'>
          <SymbolAIcon className='h-6 w-6 z-10' />
          <SymbolBIcon className='h-6 w-6 -ml-3 mt-3' />
```

Reply:
Change the margin instead of using rotate as the images are all weirdly orientated.

![image](https://user-images.githubusercontent.com/4586209/143174044-209fabe2-0bf9-41bc-8935-5c7fe1f655f8.png)

Reply:
Pls update the padding of the card to match the design too